src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -191,29 +190,6 @@ public void execute_filteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
-    @Test
```

Comment:
Should we add new test cases in the future to replace these deleted ones?

Reply:
Yes, lets keep this in mind.